### PR TITLE
chore: added inventory ui improvement

### PR DESCRIPTION
**Pull request by vishwajeetraj11**
Improvements in UI as per latest figma design

### CODE REVIEW

File: src/lib/components/v2/search/SearchBar.svelte [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
 	export let styleClass = '';
 	export let onSearchClick: (() => void) | undefined = undefined;
+	export let label: string;
 </script>
 
-<InputCard variant="search" styleClass="{styleClass} flex gap-4 items-center">
-	<Icon styleClass="opacity: .2" data={search} size={18} iconColorClass="icon-info" />
+<InputCard
+	variant="search"
+	styleClass="{styleClass} relative flex gap-4 items-center border border-[#D9DADE]"
```
Comment (souvikmishra) [inline src/lib/components/v2/search/SearchBar.svelte:19]: maybe we can use `cn` here as well

File: src/lib/components/v2/search/SearchBar.svelte [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
+		</p>
+	{/if}
+	<img src={staticImages.searchV2Icon} alt="Search Icon" class="h-6 w-6" />
 	<input
 		prefix="0x"
 		bind:value={input}
 		on:click={(e) => onSearchClick?.()}
 		id="address-display"
-		class="hideInputNumberAppearance {inputClasses.searchInputText}"
+		class={cn(inputClasses.searchInputText, 'hideInputNumberAppearance placeholder-[#A8A8A8]')}
```
Comment (souvikmishra) [inline src/lib/components/v2/search/SearchBar.svelte:34]: is `cn` needed here i.e. are we using `searchInputText` classes somewhere else also without `hideInputNumberAppearance placeholder-[#A8A8A8]`, if not then maybe we can update the `searchInputText` class itself, what do you think?

File: src/lib/components/v2/search/SearchBar.svelte [hunk 3]
```diff
@@ -1,24 +1,37 @@
 <script lang="ts">
 	import search from 'svelte-awesome/icons/search';
```
Comment (souvikmishra) [inline src/lib/components/v2/search/SearchBar.svelte:2]: maybe we can also replace the search icon with an actual svg that we can get from figma, that way we can get rid of this dependency from the project, if that is too much of a change then we can probably address it sometime later

File: src/lib/page-components/v2/oyster/inventory/OysterInventoryTableRow.svelte [hunk 4]
```diff
@@ -208,20 +191,27 @@
...
 					>
-						WITHDRAW
+						Withdraw
 					</ModalButton>
 					<ModalButton
 						variant="outlined"
 						size="small"
 						modalFor="job-amend-rate-modal-{id}"
 						disabled={isJobFinished}
+						styleClass="font-normal"
```
Comment (souvikmishra) [inline src/lib/page-components/v2/oyster/inventory/OysterInventoryTableRow.svelte:194]: does it make it easier to make the default style in the `ModalButton` and use it? as it seems like we are generally using `font-normal` as a passed in class in multiple places, maybe assess once and change accordingly

File: src/lib/utils/constants/v2/urls.ts [hunk 5]
```diff
@@ -27,6 +27,7 @@ const OYSTER_MARKETPLACE_URL = '/v2/oyster/marketplace';
 const OYSTER_OPERATOR_URL = '/v2/oyster/operator';
 const OYSTER_INVENTORY_URL = '/v2/oyster/inventory';
 const OYSTER_AUDITORS_URL = '/v2/oyster/auditors';
+const OYSTER_OWNER_HISTORY_URL = '/v2/oyster/inventory/history';
```
Comment (souvikmishra) [inline src/lib/utils/constants/v2/urls.ts:30]: i guess we can follow the convention of distinguishing routes inside oyster using `inventory` or `operator`, introducing the term `owner` might lead to confusion later, what do you think?

File: src/lib/components/v2/search/SearchBar.svelte (hunk 3 above)
Comment (vishwajeetraj11) [reply src/lib/components/v2/search/SearchBar.svelte:2]: unused import, removed.

yes, as per v2 design, will replace the search icons one by one from all components/pages.

File: src/lib/components/v2/search/SearchBar.svelte (hunk 1 above)
Comment (vishwajeetraj11) [reply src/lib/components/v2/search/SearchBar.svelte:19]: done

File: src/lib/components/v2/search/SearchBar.svelte (hunk 2 above)
Comment (vishwajeetraj11) [reply src/lib/components/v2/search/SearchBar.svelte:34]: added the placeholder inside searchInputText, it is used without hideInputNumberAppearance elsewhere, so left it as it is.

File: src/lib/page-components/v2/oyster/inventory/OysterInventoryTableRow.svelte (hunk 4 above)
Comment (vishwajeetraj11) [reply src/lib/page-components/v2/oyster/inventory/OysterInventoryTableRow.svelte:194]: okay

File: src/lib/utils/constants/v2/urls.ts (hunk 5 above)
Comment (vishwajeetraj11) [reply src/lib/utils/constants/v2/urls.ts:30]: fixed

Comment (vishwajeetraj11) [reply src/lib/utils/constants/v2/urls.ts:30]: using OYSTER_INVENTORY_HISTORY_URL as var name

**Pull request merged**